README.md

```diff
@@ -142,6 +142,7 @@ __Notes:__
 - You can use alternative authentication strategies, such as [webnative-walletauth](https://github.com/fission-codes/webnative-walletauth).
 - You can remove all traces of the user using `await session.destroy()`
 - You can load the file system separately if you're using a web worker. This is done using the combination of `configuration.fileSystem.loadImmediately = false` and `program.loadFileSystem()`
+- You can recover a file system if you've downloaded a Recovery Kit by calling `program.recoverFileSystem({ newUsername, oldUsername, readKey })`. The `oldUsername` and `readKey` can be parsed from the uploaded Recovery Kit and the `newUsername` can be generated before calling the function. Please refer to [this example](https://github.com/webnative-examples/webnative-app-template/blob/5498e7062a4578028b8b55d2ac4c611bd5daab85/src/components/auth/recover/HasRecoveryKit.svelte#L49) from Fission's Webnative App Template
```

Comment:
We may want to add a bit more context about how a recovery kit is created. If the steps are simple enough, we could list them here. Or maybe link to where we create them in WAT.

Another possibility would be adding another shorthand that makes it easy for devs to get the `username` and `readKey` so they can create recovery kits with them.

Thoughts?

Reply:
good point! I think i'll add a link to the WAT example for now. I think adding a shorthand for generating the kit will be a bit too complicated until we've ported the username scheme/hashing functionality over to webnative. What do you think?

Reply:
Sounds good! We can add that down the road. 💯